test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -951,6 +951,63 @@ function (_, $, Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthFo
           expect(test.form.securityImageTooltipText()).toEqual('This is the first time you are connecting to foo.com from this browser×');
         });
       });
+      itp('does not show anti-phishing message if security image is hidden', function () {
+        return setup({ features: { securityImage: true }})
+        .then(function (test) {
+          test.form.applyQtipCssRule();
```

Comment:
I don't think we need this... it adds a style so we can verify tooltip is hidden. But we should not test whether the tooltip is hidden or not present (which is the same for testing purposes, it is just not there)
I think we need 2 tests, and without the css rule fix you are adding, the tests will be something like:

1. hide beacon and set wrong username -> waitForBeaconChange -> verify tooltip is not there
2. wrong username -> waitForBeaconChange -> check tooltip is there -> hide beacon and resize -> waitForBeaconChange -> check tooltip is not there


And test for whether the tooltip exists, and if it does, then whether is visible or not.